index.html

```diff
@@ -0,0 +1,386 @@
+- [Ubuntu](https://help.ubuntu.com/stable/ubuntu-help/display-dual-monitors.html.en) says "With two screens, these display modes are available: Join Displays: screen edges are joined so things can pass from one display to another..."
+- [Arch Linux](https://wiki.archlinux.org/index.php/Multihead) says "Display refers to a collection of screens" and more...
+- [Chromebook Help](https://support.google.com/chromebook/answer/1060909?hl=en) says "You can connect your Chromebook to a monitor or TV" :-/
+- Android mentions "[Multi-Display](https://source.android.com/devices/tech/display/multi_display)" and "[multi-screen](https://source.android.com/devices/tech/display/multi_display/recommended-practices#foldable)"
+- [iOS/UIKit](https://developer.apple.com/documentation/uikit/windows_and_screens/displaying_content_on_a_connected_screen) says "The user can connect additional screens"
+- [Chromium code](https://source.chromium.org/chromium/chromium/src/+/master:ui/display/screen.h;l=36) says "A single Screen object exists regardless of the number of connected displays."
+
+### Multiple internal display
+
+A <dfn>multiple internal display</dfn> (MID) device is a more technical synonym to a [=dual-screen device=].
+
+### Single foldable display
+
+A <dfn>single foldable display</dfn> (SFD) refers to a [=foldable-screen device=] with a single screen.
+
+### Posture
+
+A <dfn>posture</dfn> is defined as the primary way in which the [=foldable device=] is positioned and presented to the user. Because [=foldable devices=] have a hinge they can present different [=postures=] depending on the hinge angle, device orientation, and the way on how the device is being held by the user. Example postures include 'book', 'flat', 'laptop', and 'tent'. The list of postures is expected to evolve over time, and not all foldable devices are able to support all possible postures.
+
+<div class="issue">
+We've discussed the need to rethink the names of the postures to make them as abstract as possible. The fact that 'laptop' is both an actual device and a 'posture' mode can be confusing and misleading. Can a laptop have another posture that's not laptop? Does a mobile device get into laptop posture? Similar case with tablet as a device type and posture.
+</div>
+
+### Window segments
+
+A <dfn>window segment</dfn> represents the regions (and their dimensions) of the window that reside on separate (adjacent) [=displays=].
+
+### Spanning
+
+<dfn>Spanning</dfn> is a window mode on a [=dual-screen device=], where a single application or web content window is spread across both [=screens=] of the device. The ability to maximize a given application across the two [=screens=] is enabled through the [=window manager=]. Because spanning across the two screens double the screen estate it is expected that applications and websites leverage that additional space to optimize or enhance the UX to provide a better experience.
+
+<div class="note">
```

Comment:
Perhaps this note belongs with one of the form factor definitions instead of with Spanning?

Reply:
Fixed by a984764.